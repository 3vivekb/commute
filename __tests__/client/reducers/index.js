/* globals describe, expect, it */

import rootReducer from '../../../client/reducers'

describe('reducers > root', () => {
  it('should have default state', () => {
    expect(rootReducer(undefined, { type: 'blah', payload: {} })).toMatchSnapshot()
  })
})
