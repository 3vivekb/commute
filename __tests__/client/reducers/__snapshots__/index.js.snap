exports[`reducers > root should have default state 1`] = `
Object {
  "user": Object {},
}
`;
